src/idiomatic/foundations-api-design/predictable-api/naming-conventions/01-get.md

```diff
@@ -0,0 +1,31 @@
+---
+minutes: 2
+---
```

Comment:
These are great slides about naming conventions, but I have a couple of comments. I have left the comments on the "get" and "push" slides, but please apply them everywhere. All slides in this section should follow the same structure.